src/olympia/amo/mail.py

```diff
@@ -17,30 +17,39 @@ class DevEmailBackend(BaseEmailBackend):
     `settings.SEND_REAL_EMAIL` is disabled.
 
     BUT even if `settings.SEND_REAL_EMAIL` is disabled, if the targeted
-    email address is in the `settings.EMAIL_QA_ALLOW_LIST` list,
-    the email will be sent.
+    email address is in `dev_email` group users, the email will be sent.
     """
 
+    @staticmethod
```

Comment:
Stick it as a property on the class?  (Not sure why it needs to be a method if it just returns a fixed string)

Reply:
dupe to mat's comment above